scripts/test_emscripten.sh

```diff
@@ -51,3 +51,6 @@ npm version $VERSION
 
 echo "Running solc-js tests..."
 npm run test
+
+echo "Running external tests...."
+"$REPO_DIR"/test/externalTests.sh "$REPO_DIR"/build/solc/soljson.js
```

Comment:
`REPO_DIR`? There's only `REPO_ROOT` in this script.

Could also create a variable for this up next to REPO_ROOT and use it in the "replacing soljson.js" part too.